src/build/mdn-comments.ts

```diff
@@ -1,9 +1,10 @@
-import { promises as fs } from "fs";
+import fs from "fs/promises";
 import * as path from "path";
-import { fileURLToPath } from "url";
 
-const __filename = fileURLToPath(import.meta.url);
-const __dirname = path.dirname(__filename);
+const basePath = new URL(
+  "../../inputfiles/mdn/files/en-us/web/api/",
+  import.meta.url,
+).pathname;
```

Comment:
Using pathname still requires using `path` elsewhere, but the file system API can understand URL directly.